boot.php

```diff
@@ -734,6 +734,11 @@ function set_pager_page($n) {
 
 		function init_pagehead() {
 			$interval = ((local_user()) ? get_pconfig(local_user(),'system','update_interval') : 40000);
+
+			// If the update is "deactivated" set it to the highest integer number (~24 days)
+			if ($interval < 0)
```

Comment:
is it "<1" or "-1"? In the rest of the code it's always "-1", but what happens if one enters "-2"?